recipe-server/client/control/components/recipes/RecipeDetails.js

```diff
@@ -104,14 +104,29 @@ class ArgumentsValue extends React.PureComponent {
   render() {
     const { name, value } = this.props;
 
+    let inner;
```

Comment:
I wrote the below comment first but looking at this now, I wonder if this whole section of logic that determines `inner` could be moved out to another function entirely. That may be too much abstraction when combined with the below comment, though. Open to thoughts.

Reply:
I think moving the `inner` logic into another function doesn't reduce complexity, it just shifts it to another function in the same context. I don't think that's beneficial. If there was a opportunity for re-use or simplification, I'd go for it, but I don't think there is here.